css/home.css

```diff
@@ -0,0 +1,87 @@
+html,
+body {
+  height: 83.2%;
+  font-family: "Actor", sans-serif;
+}
+
+body {
+  background-color: #4044ca;
+  margin: 0vw;
+  padding: 0vw;
+}
+
+.title {
+  color: white;
+  font-weight: bold;
+  font-size: 40px;
+  margin: 40px 5vw 10px 5vw;
+  padding: 0vw;
+}
```

Comment:
nothing will happen if you add or delete the padding, no need for it.

Reply:
Consider it Done :) thank you